packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -401,6 +401,6 @@ class Uploader {
 
 Uploader.FILE_NAME_PREFIX = 'uppy-file'
 // @todo do a proper migration to change this name
-Uploader.STORAGE_PREFIX = 'uppy-server'
+Uploader.STORAGE_PREFIX = 'companion'
```

Comment:
Do we need a proper migration to change this name as the comment says? :smile: I think this is only for temporary storage so it may not be a big deal…

Reply:
I _think_ it's not a big deal to ignore migration here, it's only used to emit progress once on active uploads and it's not a big deal if progress doesn't immediately update when an upload is restarted after a server restart … Ife probably knows better though

Reply:
> I think the worst that could happen is people will be logged out, and we have some useless keys in our Redis, or not?

True, yes this is the case. This was only vital for cases where people may have a long running file transfer and this change will immediately interrupt them. And also for resumed uploads, all the storage of upload progress will be lost.